staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -288,7 +288,7 @@ var (
 			Buckets:        requestDurationSecondsBuckets,
 			StabilityLevel: compbasemetrics.ALPHA,
 		},
-		[]string{priorityLevel, flowSchema, "type"},
+		[]string{priorityLevel, flowSchema, "type", "verb", "resource", "subresource"},
```

Comment:
does this start to cause the same cardinality issues as with the other request metric?